components/docs-chef-io/content/automate/ha_iam_user.md

```diff
@@ -38,45 +39,59 @@ Once we have an AWS account, we'll need to create an IAM user to programmaticall
 ## Creating an IAM User
 
 1. Navigate to your AWS account.
-1. Select **IAM** from the list of services from the AWS console. The **IAM dashboard** screen appears.
+2. Select **IAM** from the list of services from the AWS console. The **IAM dashboard** screen appears.
 
   {{< figure src="/images/automate/ha_aws_iam.png" alt="AWS IAM Dashboard">}}
 
-1. Select **Users** from the **Access management** menu on the left.
-1. Select **Add Users**. The **Set User Details** screen appears.
-1. Enter the user name for the new user and other necessary details.
+3. Select **Users** from the **Access management** menu on the left.
+4. Select **Add Users**. The **Set User Details** screen appears.
+5. Enter the user name for the new user and other necessary details.
 
   {{< figure src="/images/automate/ha_aws_iam_user.png" alt="AWS IAM User Creation">}}
 
-1. Check the **Access key - Programmatic access** option under the **Select AWS access type** section.
+6. Check the **Access key - Programmatic access** option under the **Select AWS access type** section.
 
 {{< figure src="/images/automate/ha_aws_iam_paccess.png" alt="AWS IAM User - Programmetic Access">}}
 
   This is the sign-in name for AWS. If you want to add multiple users, choose to Add another user for each additional user and specify their user names. You can add up to 10 users at one time.
 
   This account will be used by your AWS CLI and will be connecting to the AWS API directly by not using the Management Console.
 
-1. Select **Next: Permissions**.
-1. Select **Attach existing policies directly**.
-1. Filter the policies by keyword, `IAM`. For this user, select `IAMFullAccess` from the list of available policies. The `IAMFullAccess` policy enables this user to create and manage user permissions in AWS.
+7. Select **Next: Permissions**.
+8. Select **Attach existing policies directly**.
+9. Filter the policies by keyword, `IAM`. For this user, select `IAMFullAccess` from the list of available policies. The `IAMFullAccess` policy enables this user to create and manage user permissions in AWS.
 
 {{< figure src="/images/automate/ha_aws_iam_policy.png" alt="AWS IAM User Policy">}}
 
-1. Set the user permissions.
-1. Search for **AdministratorAccess** and select the policy.
-1. Select **Next: Tags**.
-1. Provide key name and value as tagging for the user been created.
-1. Select **Next: Review**.
+10. Set the user permissions.
```

Comment:
convert all these to 1.
as in MD these are auto incremented
Do MD file preview